index.html

```diff
@@ -10,16 +10,17 @@
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <h1 id="logo">Ari karim</h1>
+      <button id="menuu" type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
+        <span id="span1" class="span1">close</span>
+        <ul id="mobile-navv">
+          <li><a id="nav-v1" class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a id="nav-v2" class="nav-link" href="#about-me">About</a></li>
           <li>
-            <a class="nav-link" href="#contact"
+            <a id="nav-v3" class="nav-link" href="#contact"
               >Contact <i class="far fa-envelope"></i
             ></a>
           </li>
```

Comment:
**Required**

- [ ] The menu items color is different on mobile view. "You must stick to the design as much as possible" is one of the project requirements. This involves colors, margins, etc. I suggest you change the color of the nav items on the mobile view to white.

**Optional**

- It is important to always keep your code DRY. Having to repeat menu items definitely work, but I believe you can do better with your JS logic. I'll you make research on some JS windows APIs like `window.innerWidth`, `document.body.clientWidth`, etc that allows you to access the current screen size of your app, then use it to change the text direction/flex properties of the menu items. This is a better way to ensure that there's no repetition of code because it means that you're just changing the text direction based on the screen size.

Reply:
Done

Reply:
This issue was addressed successfully as well :heavy_check_mark:

Reply:
- [ ] [Optional] Just an optional requirement would be to make the navbar full screen as it is given in the mockup